.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,26 @@
+name: golangci-lint
+
+on:
+  push:
+    branches: [ main ]
+  pull_request: {}
+
+permissions:
+  # https://docs.github.com/en/rest/overview/permissions-required-for-github-apps?apiVersion=2022-11-28#repository-permissions-for-contents
+  contents: read
+
+jobs:
+  go:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions/setup-go@v4
+      with:
+        go-version: stable
+    - name: golangci-lint
+      uses: golangci/golangci-lint-action@v3
+      with:
+        version: latest
+        # Enable the gosec linter w/o having to create a .golangci.yml config
+        args: -E gosec
+        only-new-issues: true
```

Comment:
I would just add this (and the `permission:`, I wasn't aware that this existed so far) to the existing `go` job in `go.yml`. Both workflows don't take that long that we gain much by running them in parallel.